packages/react/src/Token/AvatarToken.module.css

```diff
@@ -0,0 +1,37 @@
+:root {
+  --spacing: calc(var(--base-size-4) * 2);
+}
+
+.AvatarContainer {
+  display: block;
+}
+
+/* TODO: Remove this once the avatar component is converted to CSS modules */
+.Avatar {
+  width: 100% !important;
+  height: 100% !important;
+}
+
+.Token {
+  padding-left: var(--base-size-4) !important;
+}
+
+.AvatarContainer[data-size='small'] {
```

Comment:
```suggestion
.AvatarContainer:where([data-size='small']) {
```

Reply:
Hi @jonrohan, I'll get these changes in. Quick question though, why is `:where` preferred over the bare attribute selector?

Reply:
Where will reduce the specificity on the attribute selector and make it easier to override the styles in dotcom. We try and add it whenever the attribute lands on the same dom node where we add the passed in className